cfgov/core/management/commands/watchserver.py

```diff
@@ -0,0 +1,39 @@
+import os
+import subprocess
+import atexit
+import signal
+
+from django.conf import settings
+from django.contrib.staticfiles.management.commands.runserver import Command\
+    as StaticfilesRunserverCommand
+
+
+# Inspired by:
+# https://lincolnloop.com/blog/simplifying-your-django-frontend-tasks-grunt/
+# Extended to potentially support multiple grunt-watched directories.
+
+class Command(StaticfilesRunserverCommand):
+
+    def inner_run(self, *args, **options):
+        self.start_gulp()
+        return super(Command, self).inner_run(*args, **options)
+
+    def start_gulp(self):
+        self.stdout.write('>>> Starting grunt in %s' % dir)
+        sub = subprocess.Popen(
+            'gulp watch',
+            shell=True,
+            stdin=subprocess.PIPE,
+            stdout=self.stdout,
+            stderr=self.stderr,
+        )
+
+        self.gulp_process=sub
+
+        self.stdout.write('>>> Gulp process on pid {0}'.format(sub.pid))
+
+        def kill_gulp_process(sub):
+            self.stdout.write('>>> Closing grunt process %s' % sub.pid)
```

Comment:
`gulp` not `grunt`. Maybe pull this out into a variable if this setup might be used other places in the bureau that may be using one or the other.